## ğŸ“… Day 38: Lidando com exceÃ§Ãµes â€“ DivisÃ£o por Zero

### ğŸ§© Desafio  
Escreva uma funÃ§Ã£o que realize uma divisÃ£o entre dois nÃºmeros e **trate a exceÃ§Ã£o** de **divisÃ£o por zero**, exibindo uma mensagem amigÃ¡vel ao usuÃ¡rio.


### ğŸ“š Aprendizados

- Uso do bloco `try` / `except`
- Tratamento de exceÃ§Ãµes especÃ­ficas
- SeguranÃ§a e estabilidade em operaÃ§Ãµes aritmÃ©ticas
- UtilizaÃ§Ã£o de um tratamento de erro nativo no python (`ZeroDivisionError`)
